ajax/libs/jQuery.Gantt/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jQuery.Gantt",
+  "version": "1.2.2",
+  "description": "jQuery Gantt Chart is a simple chart that implements gantt functionality as a jQuery component.",
```

Comment:
Should the last word "component" be "plugin" according to https://mbielanczuk.com/jquery-gantt/?